src/main/resources/edu/cornell/kfs/module/purap/document/validation/configuration/PurchaseOrderValidation.xml

```diff
@@ -25,9 +25,7 @@
   		<property name="validations">
   			<list>  				
  				<!-- KFSPTS-2096 -->
-				<bean parent="Purchasing-saveItemValidation" scope="prototype">								
-					<property name="quitOnFail" value="false" />
-				</bean>  										
+				<bean parent="Purchasing-saveItemValidation" scope="prototype" p:quitOnFail="true" />								 										
```

Comment:
The old XML was setting "quitOnFail" to false, whereas this revised XML is setting "quitOnFail" to true. Can you please confirm whether the new behavior is the correct option?

Reply:
Good catch